src/pages/onboarding/[collectionSlug].tsx

```diff
@@ -0,0 +1,62 @@
+import { GetStaticPropsContext } from 'next'
+import { getCollectionsBySection } from 'lib/learn-client/api/collection'
+import { Collection as ApiCollection } from 'lib/learn-client/types'
+import { stripUndefinedProperties } from 'lib/strip-undefined-props'
+import { splitProductFromFilename } from 'views/tutorial-view/utils'
+import OnboardingCollectionView from 'views/onboarding/collection-view'
+import onboardingData from 'data/onboarding.json'
+import { OnboardingCollectionViewProps } from 'views/onboarding/types'
+
+export async function getStaticProps({
+	params,
+}: GetStaticPropsContext<{ collectionSlug: string }>): Promise<{
+	props: OnboardingCollectionViewProps
+}> {
+	const allWafCollections = await getCollectionsBySection(onboardingData.slug)
+	const currentCollection = allWafCollections.find(
+		(collection: ApiCollection) =>
+			collection.slug === `${onboardingData.slug}/${params.collectionSlug}`
+	)
+
+	const breadcrumbLinks = [
+		{ title: 'Developer', url: '/' },
+		{ title: onboardingData.name },
```

Comment:
Should I include this even though there is no landing page? 💭

Reply:
I agree! I think the only potential concern here would be that this isn't a link whereas so far all of our breadcrumbs have been links. This doesn't necessarily say they have to be links though: https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/

Reply:
🤔 I could remove it to remain consistent with our other views.